esque/messages/avromessage.py

```diff
@@ -72,7 +73,14 @@ def __init__(self, schema_registry_client):
 
     def write_message(self, message: Message):
         _, _, _, plaintext_message = self.avro_decoder.decode_message_from_avro(message)
-        click.echo(json.dumps(plaintext_message))
+        click.echo(json.dumps(plaintext_message, default=StdOutAvroWriter.deserializer))
+
+    @staticmethod
+    def deserializer(value):
+        if isinstance(value, (datetime, date)):
+            return value.isoformat()
+        elif isinstance(value, bytes):
+            return "0x{}".format(value.hex())
```

Comment:
How about using base64? That will be a lot shorter for larger messages...  
https://docs.python.org/3/library/base64.html#module-base64

Reply:
Sure, why not.